contracts/oracle/src/contract/alarms.rs

```diff
@@ -25,14 +26,23 @@ use crate::{
 
 pub type AlarmReplyId = u64;
 
-pub struct MarketAlarms;
+pub struct MarketAlarms(AlarmReplyId);
 
 impl MarketAlarms {
     const PRICE_ALARMS: PriceAlarms<'static> =
         PriceAlarms::new("alarms_below", "index_below", "alarms_above", "index_above");
 
     const MSG_ID: Item<'static, AlarmReplyId> = Item::new("msg_id");
 
+    pub fn load(storage: &dyn Storage) -> Result<Self, ContractError> {
+        let next_id = Self::MSG_ID.may_load(storage)?.unwrap_or_default();
+        Ok(MarketAlarms(next_id))
+    }
+
+    pub fn save(&self, storage: &mut dyn Storage) -> Result<(), ContractError> {
+        Ok(Self::MSG_ID.save(storage, &self.0)?)
+    }
+
```

Comment:
why do we store the reply id? After all, it should be unique in the scope of a single transaction.

Reply:
after consulting [this](https://docs.cosmwasm.com/docs/smart-contracts/message/submessage) and [this](https://github.com/CosmWasm/cosmwasm/blob/32f308a1a56ae5b8278947891306f7a374c3df94/packages/std/src/results/submessages.rs#L32) it seems that they even not need to be unique at all. 

So please remove the code that stores it. I believe a single constant would suffice.